features.txt

```diff
@@ -108,6 +108,10 @@ json-parse-with-source
 # https://github.com/tc39/proposal-iterator-helpers
 iterator-helpers
 
+# Promise.withResolvers
+# https://github.com/tc30/proposal-promise-with-resolvers
```

Comment:
Noticed a typo here - the link has `tc30` instead of `tc39`

Reply:
```suggestion
# https://github.com/tc39/proposal-promise-with-resolvers
```